integrations/airflow/test-requirements.txt

```diff
@@ -5,16 +5,16 @@ apache-airflow[google]==1.10.12
 apache-airflow[postgres]==1.10.12
 codecov==2.1.8
 marquez-python==0.7.12
-pytest==6.1.2
+pytest
 pytest-cov==2.10.1
 requests==2.24.0
-psycopg2-binary==2.8.6
+psycopg2-binary
 mock>=4.0.2
 httplib2>=0.18.1
-google-cloud-bigquery>=1.28.0
-google-auth-httplib2>=0.0.4
-google-api-core>=1.22.2
-google-api-python-client>=1.12.2
+google-cloud-bigquery
+google-auth-httplib2
+google-api-core
+google-api-python-client
 pandas-gbq>=0.13.2
-google-cloud-storage>=1.31.2
+google-cloud-storage
```

Comment:
should these versions be fixed?

Reply:
Ideally, yes. But, CI was complaining that the artifacts aren't in pypi:

```
ERROR: Could not find a version that satisfies the requirement google-cloud-bigquery>=1.28.0
ERROR: No matching distribution found for google-cloud-bigquery>=1.28.0
```

Removing the version "fixed" the issue, but certainly not ideal.